libnethack/src/shknam.c

```diff
@@ -271,7 +235,7 @@ nameshk(struct monst *shk, const char *const *nlp, struct level *lev)
                     continue;
                 continue;       /* next `trycnt' iteration */
             } else {
-                shname = shk->female ? "Lucrezia" : "Dirk";
+                shname = shk->female ? "Paul Mezzanini" : "Matt Campbell";
```

Comment:
Shush you. 

If you have a good candidate for a female that can occur often let me know, but until that happens Paul is female. 

> On Dec 29, 2015, at 15:30, Ross Delinger <notifications@github.com> wrote:
> 
> In libnethack/src/shknam.c:
> 
> > @@ -271,7 +235,7 @@ nameshk(struct monst *shk, const char *const *nlp, struct level *lev)
> >                      continue;
> >                  continue;       /* next `trycnt' iteration */
> >              } else {
> > -                shname = shk->female ? "Lucrezia" : "Dirk";
> > +                shname = shk->female ? "Paul Mezzanini" : "Matt Campbell";
> Shouldn't one of these be a female name?
> 
> ―
> Reply to this email directly or view it on GitHub.
>